scripts/test-consolidated-server.js

```
@@ -0,0 +1,279 @@
+// Test 5: Check for potential conflicts
+function testForConflicts() {
+  console.log('\n⚠️  Testing for potential conflicts...');
+  
+  let warnings = [];
+  
+  try {
+    // Check if standalone server is still being used in development
+    const packageJson = JSON.parse(readFileSync('package.json', 'utf8'));
+    
+    if (packageJson.scripts['dev:backend'] && packageJson.scripts['dev:backend'].includes('server/index.js')) {
+      warnings.push('Development still uses standalone server (server/index.js)');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)